packages/core/EventManager.ts

```diff
@@ -98,10 +98,15 @@ export default class EventManager {
     // This includes all calendar-related apps, traditional calendars such as Google Calendar
     // (type google_calendar) and non-traditional calendars such as CRMs like Close.com
     // (type closecom_other_calendar)
-    this.calendarCredentials = appCredentials.filter(
-      // Backwards compatibility until CRM manager is implemented
-      (cred) => cred.type.endsWith("_calendar") && !cred.type.includes("other_calendar")
-    );
+    this.calendarCredentials = appCredentials
+      .filter(
+        // Backwards compatibility until CRM manager is implemented
+        (cred) => cred.type.endsWith("_calendar") && !cred.type.includes("other_calendar")
+      )
+      //see https://github.com/calcom/cal.com/issues/11671#issue-1923600672
+      .sort((a, b) => {
+        return b.id - a.id;
+      });
```

Comment:
how about we name this arrow function, `latestFirst` and reuse it  below as well. 
Makes the code more readable

Reply:
Do you mean `oldestFirst` as we are sorting in ascending order

Reply:
I think we are sorting in descending order only, unless I am misunderstanding

![image](https://github.com/calcom/cal.com/assets/1780212/a6505ae8-cab5-48fe-9b8f-0be785f80c0d)